<ui version="4.0" stdsetdef="1">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>SnapshotSettingsPanel</class>
  <widget class="QWidget" name="SnapshotSettingsPanel">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>506</width>
        <height>301</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>LDView Snapshot Options</string>
    </property>
    <property name="windowIcon">
      <iconset resource="resources.qrc" >
      <normaloff>:/images/images/LDViewIcon16.png</normaloff>:/images/images/LDViewIcon16.png</iconset>
    </property>
    <layout class="QVBoxLayout">
      <property name="margin">
        <number>11</number>
      </property>
      <property name="spacing">
        <number>4</number>
      </property>
      <item>
        <layout class="QVBoxLayout">
          <property name="margin">
            <number>11</number>
          </property>
          <property name="spacing">
            <number>4</number>
          </property>
          <item>
            <layout class="QGridLayout">
              <property name="margin">
                <number>0</number>
              </property>
              <property name="spacing">
                <number>4</number>
              </property>
              <item row="0" column="0">
                <widget class="QGroupBox" name="seriesEnabledButton">
                  <property name="title">
                    <string>Save snapshot series</string>
                  </property>
                  <property name="checkable">
                    <bool>true</bool>
                  </property>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>11</number>
                    </property>
                    <property name="spacing">
                      <number>4</number>
                    </property>
                    <item>
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <widget class="QLabel" name="digitLabel">
                            <property name="text">
                              <string>Number of Digits:</string>
                            </property>
                            <property name="wordWrap">
                              <bool>false</bool>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <widget class="QSpinBox" name="digitBox">
                            <property name="maximum">
                              <number>4</number>
                            </property>
                            <property name="minimum">
                              <number>1</number>
                            </property>
                            <property name="value">
                              <number>1</number>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <spacer name="Spacer_digitbox">
                            <property name="sizeHint">
                              <size>
                                <width>20</width>
                                <height>20</height>
                              </size>
                            </property>
                            <property name="sizeType">
                              <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                          </spacer>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QGroupBox" name="sizeEnabledButton">
                  <property name="title">
                    <string>Save image with specific size</string>
                  </property>
                  <property name="checkable">
                    <bool>true</bool>
                  </property>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>11</number>
                    </property>
                    <property name="spacing">
                      <number>4</number>
                    </property>
                    <item>
                      <layout class="QHBoxLayout">
                        <property name="margin">
                          <number>0</number>
                        </property>
                        <property name="spacing">
                          <number>0</number>
                        </property>
                        <item>
                          <layout class="QVBoxLayout">
                            <property name="margin">
                              <number>0</number>
                            </property>
                            <property name="spacing">
                              <number>4</number>
                            </property>
                            <item>
                              <widget class="QLabel" name="widthLabel">
                                <property name="text">
                                  <string>Width:</string>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QLabel" name="heightLabel">
                                <property name="text">
                                  <string>Height:</string>
                                </property>
                                <property name="wordWrap">
                                  <bool>false</bool>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <layout class="QVBoxLayout">
                            <property name="margin">
                              <number>0</number>
                            </property>
                            <property name="spacing">
                              <number>4</number>
                            </property>
                            <item>
                              <widget class="QSpinBox" name="widthBox">
                                <property name="maximum">
                                  <number>9999</number>
                                </property>
                                <property name="minimum">
                                  <number>50</number>
                                </property>
                                <property name="value">
                                  <number>1024</number>
                                </property>
                              </widget>
                            </item>
                            <item>
                              <widget class="QSpinBox" name="heightBox">
                                <property name="maximum">
                                  <number>9999</number>
                                </property>
                                <property name="minimum">
                                  <number>50</number>
                                </property>
                                <property name="value">
                                  <number>768</number>
                                </property>
                              </widget>
                            </item>
                          </layout>
                        </item>
                        <item>
                          <spacer name="Spacer16_4">
                            <property name="sizeHint">
                              <size>
                                <width>16</width>
                                <height>20</height>
                              </size>
                            </property>
                            <property name="sizeType">
                              <enum>QSizePolicy::Fixed</enum>
                            </property>
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                          </spacer>
                        </item>
                        <item>
                          <widget class="QCheckBox" name="zoomtofitEnabledButton">
                            <property name="text">
                              <string>Zoom to Fit</string>
                            </property>
                          </widget>
                        </item>
                        <item>
                          <spacer name="Spacer24">
                            <property name="sizeHint">
                              <size>
                                <width>20</width>
                                <height>20</height>
                              </size>
                            </property>
                            <property name="sizeType">
                              <enum>QSizePolicy::Expanding</enum>
                            </property>
                            <property name="orientation">
                              <enum>Qt::Horizontal</enum>
                            </property>
                          </spacer>
                        </item>
                      </layout>
                    </item>
                  </layout>
                </widget>
              </item>
              <item row="0" column="2">
                <widget class="QGroupBox" name="pbufferBox">
                  <property name="title">
                    <string>Misc</string>
                  </property>
                  <layout class="QVBoxLayout">
                    <property name="margin">
                      <number>11</number>
                    </property>
                    <property name="spacing">
                      <number>4</number>
                    </property>
                    <item>
                      <widget class="QCheckBox" name="transparentBackgroundButton">
                        <property name="text">
                          <string>Transparent Background</string>
                        </property>
                      </widget>
                    </item>
                    <item>
                      <widget class="QCheckBox" name="autoCropButton">
                        <property name="text">
                          <string>Auto-crop</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>
              <item row="0" column="3">
                <widget class="QGroupBox" name="allStepsBox">
                  <property name="title">
                    <string>All Steps</string>
                  </property>
                  <property name="checkable">
                    <bool>true</bool>
                  </property>
                  <layout class="QGridLayout">
                    <property name="margin">
                      <number>11</number>
                    </property>
                    <property name="spacing">
                      <number>4</number>
                    </property>
                    <item row="0" column="0">
                      <widget class="QLabel" name="suffixLabel">
                        <property name="text">
                          <string>Suffix:</string>
                        </property>
                        <property name="wordWrap">
                          <bool>false</bool>
                        </property>
                      </widget>
                    </item>
                    <item row="0" column="1">
                      <widget class="QLineEdit" name="suffixEdit"/>
                    </item>
                    <item row="1" column="1">
                      <widget class="QCheckBox" name="sameScaleCheck">
                        <property name="text">
                          <string>Same Scale</string>
                        </property>
                      </widget>
                    </item>
                  </layout>
                </widget>
              </item>
            </layout>
          </item>
          <item>
            <spacer name="Spacer10_2">
              <property name="sizeHint">
                <size>
                  <width>20</width>
                  <height>20</height>
                </size>
              </property>
              <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="orientation">
                <enum>Qt::Vertical</enum>
              </property>
            </spacer>
          </item>
        </layout>
      </item>
    </layout>
  </widget>
  <layoutdefault spacing="4" margin="11"/>
  <tabstops>
    <tabstop>seriesEnabledButton</tabstop>
    <tabstop>digitBox</tabstop>
    <tabstop>sizeEnabledButton</tabstop>
    <tabstop>widthBox</tabstop>
    <tabstop>heightBox</tabstop>
    <tabstop>zoomtofitEnabledButton</tabstop>
  </tabstops>
  <pixmapfunction></pixmapfunction>
  <resources>
   <include location="resources.qrc" />
  </resources>
</ui>
